integration-tests/sanity/sanity_test.go

```diff
@@ -39,7 +39,7 @@ var _ = Describe("Sanity", func() {
 	Context("should create cstor volume", func() {
 		It("should apply the cstor pvc", func() {
 			// Getting the PVC artifact
-			artifact, err := artifacts.GetArtifactUnstructured(artifacts.CStorPVCArtifacts)
+			artifact, err := artifacts.GetArtifactUnstructuredFromFile(artifacts.CStorPVCArtifacts)
```

Comment:
Mark a TODO as these things should not be in artifacts yaml.
Only the openebs operator contents will reside in artifacts yaml

Reply:
added